nautobot/core/settings.py

```diff
@@ -1082,3 +1082,5 @@ def silk_request_logging_intercept_logic(request):
 
 
 SILKY_INTERCEPT_FUNC = silk_request_logging_intercept_logic
+
+NAUTOBOT_EVENT_BROKERS = {}
```

Comment:
I'd probably just call this `EVENT_BROKERS` and move it up to around line 98 of this file to be alphabetically inline. Don't forget to also:

- Add as a commented-out and documented entry in `nautobot/core/templates/nautobot_config.py.j2` around line 362
- Add documentation in `nautobot/core/settings.yaml`